jcl/src/java.base/share/classes/java/lang/invoke/MethodHandleResolver.java

```diff
@@ -129,21 +125,41 @@ private static final Object resolveConstantDynamic(long j9class, String name, St
 		 */
 		if (bsmTypeArgCount < 1 || MethodHandles.Lookup.class != bsm.type().parameterType(0)) {
 			/*[MSG "K0A01", "Constant_Dynamic references bootstrap method '{0}' does not have java.lang.invoke.MethodHandles.Lookup as first parameter."]*/
+/*[IF OPENJDK_METHODHANDLES]*/
+			throw new BootstrapMethodError(Msg.getString("K0A01", Lookup.IMPL_LOOKUP.revealDirect(bsm).getName())); //$NON-NLS-1$
+/*[ELSE] OPENJDK_METHODHANDLES*/
 			throw new BootstrapMethodError(Msg.getString("K0A01", bsm.getMethodName())); //$NON-NLS-1$
+/*[ENDIF] OPENJDK_METHODHANDLES*/
```

Comment:
This is an error path that is very very unlikely to happen in production.  Use the common code sequence for both OJ9 & OJDK implementations
```suggestion
			throw new BootstrapMethodError(Msg.getString("K0A01", Lookup.IMPL_LOOKUP.revealDirect(bsm).getName())); //$NON-NLS-1$
```

Reply:
Update to use the common code sequence.